program/check-upper-case/CheckUpperCase.ts

```diff
@@ -0,0 +1,8 @@
+
+function checkUpperCaseInJS(character: string) {
```

Comment:
Same comment for JS file. Could make it `checkUpperCase` as the function name instead.